backend/src/review/review.service.ts

```diff
@@ -16,9 +17,27 @@ type ReviewQuery = {
 
 @Injectable()
 export class ReviewService {
-  constructor(private readonly databaseService: DatabaseService) {}
+  constructor(
+    private readonly databaseService: DatabaseService,
+    private reviewSimilarityService: ReviewSimilarityService,
+  ) {}
+
+  private async getExistingRevewsForUser(
+    inputReview: Review,
+  ): Promise<Review[]> {
+    try {
+      return await this.databaseService.sql<Review[]>`SELECT REVIEW
+        FROM review
+        WHERE landlord = ${inputReview.landlord}
+          AND ZIP = ${inputReview.zip};`;
```

Comment:
This line as well, should be `AND zip = ${inputReview.zip.toLocaleUpperCase()}`

Reply:
![image](https://github.com/RateTheLandlord/RateTheLandlord/assets/47609209/4efa593e-ab25-481c-94a8-af851023d0d0)

Resolved :)